package.json

```diff
@@ -10,8 +10,8 @@
   },
   "scripts": {
     "build": "rimraf lib && tsc -p tsconfig.json",
-    "lint": "prettier --check 'src/**/*.ts' 'test/**/*.ts' 'docs/*.md' *.md package.json tsconfig.json",
-    "lint:fix": "prettier --write 'src/**/*.ts' 'test/**/*.ts' 'docs/*.md' *.md package.json tsconfig.json",
+    "lint": "prettier --check \"src/**/*.ts\" \"test/**/*.ts\" \"docs/*.md\" *.md package.json tsconfig.json --end-of-line auto",
+    "lint:fix": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"docs/*.md\" *.md package.json tsconfig.json --end-of-line auto",
```

Comment:
Is windows choking on the single quotes? 

I wonder about the `--end-of-line` setting. The default should be `lf` which is just `\n` if I understand correctly? That's what it should be. I'm just curious because I'm using the same prettier scripts in many projects, so I'd like to better understand the problem windows users might have

Reply:
Yes it is choking on the single quotes in the command line, they are not supported for what ever reason.

The end-of-line thing may depend on the user. I just checked out the git repository with the default settings in windows and then it uses auto line endings in git (I did not change any settings in git). That leads to \r\n in all the files and \n in the checked in files.
I know that this can create problems in some cases but I personally never had any that could not be fixed easily.
When I run prettier it changes all the line endings to \n and then git tells me that everything changed.
I did not see any remarks about using forced \n on git checkout so I though this is gives better flexibility.

Reply:
that is all very odd. But maybe we should change it to `--end-of-line lf` then?

Reply:
--end-of-line lf is the default.

The whole thing is kind of a windows problem.
Git windows installer default is autocrlf=true which is probably not the best setting but helps with editors. The git default is false.
So it would probably be useful to document somewhere that the linter wants lf and windows user should make sure that autocrlf=false is set for the clone.
Otherwise "auto" in the linter should handle the line ending good enough. I personally don't think the line ending is that important as long as it does not change on every commit :-)

Reply:
setting `* text=auto eol=lf` also not a bad solution
https://prettier.io/docs/en/options.html#end-of-line

Reply:
all right, thank you! I don't 100% understand it but if it saves Windows users some headache that by all means, let's do it :) Thank you for the explanations!